apps/codeforafrica/content/pages/about.md

```diff
@@ -69,6 +69,9 @@ our-partners:
     - aa280d47-ee0c-4cf7-93aa-1fbe1a087439
     - 1079f9ae-0fbd-472b-bbfb-518145c28b1d
     - daa2a622-0277-44be-b302-dbc703f89522
+  action:
+    href: /about/partners
+    content: Meet our Partners
```

Comment:
About page shows all partners (similar to /about/partners) so this shouldn't be there.